src/stores/pacbioRunCreate.js

```diff
@@ -178,8 +205,46 @@ export const usePacbioRunCreateStore = defineStore('pacbioRunCreate', {
 
```

Comment:
We can replace references to `runItem` and `runTypeItem` with `state.run` and `state.runItem` and remove line 204 and 206